app.js

```diff
@@ -108,6 +151,10 @@ bot.on('text', (ctx) => {
                         reply += capitalizeFirstLetter(question.answer_key) + " - " + user[question.answer_key] + "\n";
                     })
                     utility.sendMessage(ctx, reply, 'introductory', 'group');
+                } else if (user && user.last_asked == constants.questions.length + 1) {
+                    var mess = "Dear " + from.first_name
+                    + ". Please try to run this command in a private chat."
```

Comment:
I guess it's better to move it to constants.js.

Reply:
WIll move a lot of these there.